.github/workflows/release.yml

```diff
@@ -21,6 +21,9 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+    - uses: extractions/setup-just@aa5d15c144db4585980a44ebfdd2cf337c4f14cb
+      with:
+        just-version: 1.2.0
```

Comment:
FWIW, i think it would be fine to omit the pinned version, since we don't have any dependabot-like ability to keep this updated